pydantic/fields.py

```diff
@@ -198,9 +200,12 @@ def _populate_sub_fields(self) -> None:  # noqa: C901 (ignore complexity)
 
         if issubclass(origin, Tuple):  # type: ignore
             self.shape = Shape.TUPLE
-            self.sub_fields = [
-                self._create_sub_type(t, f'{self.name}_{i}') for i, t in enumerate(self.type_.__args__)  # type: ignore
-            ]
+            self.sub_fields = []
```

Comment:
in the case of an ellipsis, can't we use 

```py
self.type_ = self.type_.__args__[0]
```
like we do for list or sequence.

Since I believe only one type can be supplied for tuples with variable length, the docs [aren't very clear on this](https://docs.python.org/3/library/typing.html#typing.Tuple).

Then you might be able to avoid having to adding `has_ellipsis`